docs/proposals/prop-0001-proposals.md

```diff
@@ -0,0 +1,163 @@
+feedback.
+
+We intend proposals to be the primary mechanism for proposing major new features, for collecting team and community 
+input on an issue, and for documenting the design decisions that have gone into anonlink. A proposal should provide
+a concise technical specification of the feature or change and the rationale behind it.
+
+The author is responsible for building consensus and documenting dissenting opinions.
+
+The audience of these proposals are developers of anonlink itself. Proposals are maintained as text files in the
+versioned repository, their revision history is the historical record of the feature proposal. 
+
+## Motivation 
+
+
+## Workflow
+
+The more focused the proposal, the more successful it tends to be. If in doubt, split into several well-focused ones.
+
+Each proposal must have a champion — someone who writes the proposal using the style and format described below, 
+shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. 
+
+The proposal should be submitted as a draft proposal via a GitHub pull request to the `docs/proposals` directory with 
+the name prop-<n>-<short-title>.md where <n> is an appropriately assigned 4 digit number 
+(e.g., prop-0000-fooDescription.rst). The draft must comply with the technical specifications outlined below.
+
+Once the PR for the proposal is in place, a post should be made to the mailing list containing the abstract and a link
+to the PR, with the purpose of starting discussion on usage and impact. Discussion on the pull request will have a 
+broader scope, also including details of implementation.
+
+At the earliest convenience, the PR should be merged (regardless of whether it is accepted during discussion). 
+Additional PRs may be made by the Author/s to update or expand the proposal, or by maintainers to set its status, and 
+links to discussion URLs, etc.
+
+It is generally recommended that a *prototype* implementation be co-developed with the proposal, as ideas that sound 
+good in principle sometimes turn out to be impractical when subjected to the test of implementation. Often it 
+makes sense for the prototype implementation to be made available as a separate PR to the repo (making sure to 
+appropriately mark the PR as a WIP). Another option is to include a prototype as an auxiliary file along with the 
+proposal. 
+
+## Review and Resolution
+
+Proposals are discussed on the anonlink mailing list. The possible paths of the status of a proposal are:
+
+![](prop-0001/resolution-process.png)
+
+Eventually, after discussion, there may be a consensus that the proposal should be accepted – see the next section 
+for details. At this point the status becomes `Accepted`.
+
+Once a Proposal has been `Accepted`, the implementation or change must be completed (incorporated into the main source 
+code repository) before updating the status of the proposal to `Final`.
```

Comment:
Would it be better to set a deadline when the proposal is set to Accepted? An Epic might be needed in our sprint board as well?

Reply:
I'm trying to separate out the Data61 internal process of prioritization and planning and our sprint board and make this transparent for any open source collaborators. I'm unsure having a deadline helps